src/sentry/models/files/control_fileblob.py

```diff
@@ -12,3 +14,19 @@ class Meta:
 
     FILE_BLOB_OWNER_MODEL = ControlFileBlobOwner
     DELETE_FILE_TASK = delete_file_control
+
+    # TODO(hybrid-cloud): This is a temporary measure to allow concurrent production deployments of filestore
+    # We override the behavior of only the control silo file storage implementation to use
+    # the new control instance while production runs in monolith mode
+    @classmethod
+    def _storage_config(cls):
+        from sentry import options as options_store
+
+        config = None
+        try:
+            # If these options exist, use them. Otherwise fallback to default behavior
+            config["backend"] = options_store.get("filestore.backend.control")
+            config["options"] = options_store.get("filestore.options.control")
```

Comment:
Should these options have `control` first to act as a namespace?

Reply:
No strong preference here. Is that a pattern we're doing? 

Personally I would've liked to reverse the names on all these models and whatnot too so all the FileBlob stuff was adjacent...but we don't really do that.